src/web/mjs/connectors/sakuramanga.mjs

```diff
@@ -0,0 +1,82 @@
+import Connector from '../engine/Connector.mjs';
+import Manga from '../engine/Manga.mjs';
+
+export default class SakuraManga extends Connector {
+
+    constructor() {
+        super();
+        super.id = 'sakuramanga';
+        super.label = 'Sakura Manga';
+        this.tags = ['manga', 'japanese'];
+        this.url = 'https://sakuramanga.net';
+
+        this.list = '/japanese-manga-list';
+    }
+
+    async _getMangaFromURI(uri) {
+        try {
+            let request = new Request(uri, this.requestOptions);
+            let data = await this.fetchDOM(request, 'head title');
+            let id = uri.pathname;
+            let title = data[0].text.split(' | ')[0].trim();
```

Comment:
Title seems pretty long for being stripped down already, is this intended?

<img width="277" alt="copy-paste" src="https://user-images.githubusercontent.com/27136773/65079660-fa11aa80-d99f-11e9-862f-38f907ec731e.png">

Reply:
Since its Japanese manga i figured the full title would be best.
but what you got there is indeed incorrect
![image](https://user-images.githubusercontent.com/36196198/65080331-51644a80-d9a1-11e9-9023-32432c3676e4.png)

will review it

Reply:
Will create bug for this after merging the pull request